include/mercury/kw/kw.h

```diff
@@ -34,25 +35,38 @@ class KanoWorld
      */
     explicit KanoWorld(std::shared_ptr<IHTTPClient> client =
         std::make_shared<HTTPClient>());
-    bool login(string username, string password, bool verbose);
-    bool refresh_token(string token, bool verbose);
+
+    bool login(const string& username, const string& password,
+               const bool verbose = false);
+    bool logout(const bool verbose = false);
+    bool refresh_token(string token, const bool verbose = false);
 
     string get_hostname(string config_filename);
     string get_refresh_header(string token);
 
-    static size_t write_function(void *ptr, size_t size, size_t nmemb, void *stream);
+    bool is_logged_in(const bool verbose = false);
+    string whoami();
 
-    bool am_i_logged_in(void);
-    string whoami(void);
 
     /**
      * \brief Determine if the user's account has been verified with parental
      *        permission.
      */
     bool is_account_verified();
 
+    const string data_filename;
```

Comment:
Do we want this to be a public member? Can it be private?

Reply:
Looks like @skarbat is using this for something in the tests so don't think that it can become private right now.